src/beautifiers/rustfmt.coffee

```diff
@@ -34,7 +34,7 @@ module.exports = class Rustfmt extends Beautifier
     else
       @run(program, ["--version"], help: help)
         .then((stdout) ->
-          if /^0\.(?:[0-4]\.[0-9])/.test(stdout.trim())
+          if /^0\.(?:[0-4]\.[0-9])(?!-nightly)/.test(stdout.trim())
```

Comment:
I would love to have a look. Cannot promise I'll have enough free cycles to implement it though.